### PR TITLE
Added a print within an if such that the compiler is hopefully less likely to optimize it away

### PR DESCRIPTION
**Pull request by Mads518d**

### CODE REVIEW

File: benchmarks/n-body/c/bench.c [hunk 1]
```diff
@@ -213,12 +213,26 @@ int main(int argc, char *argv[]) { // Modified to work with RAPL and with out be
         v[0] = _mm256_mul_pd(o, _mm256_set1_pd(-1.0 / SOLAR_MASS));
 
 
-        //printf("%.9f\n", energy(m, p, v));
+        //
+        // Det følgende er ændret
```
Comment (Jakobzs) [inline benchmarks/n-body/c/bench.c]: INGEN KOMMENTARER PÅ DANSK

Comment (Jakobzs) [inline benchmarks/n-body/c/bench.c]:
```suggestion
        // The following has been changed
```

File: benchmarks/n-body/csharp/Program.cs [hunk 2]
```diff
@@ -206,6 +207,9 @@ static double Energy(double* m, V256d* p, V256d* v)
 		}
 	}
 
+
```
Comment (Jakobzs) [inline benchmarks/n-body/csharp/Program.cs]: Remove two newlines I think

**Review by Jakobzs — CHANGES_REQUESTED** [review]

File: benchmarks/n-body/java/Bench.java [hunk 3]
```diff
@@ -63,10 +63,20 @@ public final class nbody {
         public static void N_Body(int n) {
 
             NBodySystem bodies = new NBodySystem();
-            //System.out.printf("%.9f\n", bodies.energy());
+
+            double energy1 = bodies.energy();
```
Comment (Villiamfj) [inline benchmarks/n-body/java/Bench.java:67]: Kunne godt bruge en comment med noget "stopping compiler optimisation"

File: benchmarks/n-body/cpp/bench.cpp [hunk 4]
```diff
@@ -253,9 +253,21 @@ int main(int argc, char* argv[]) {
         start_rapl();
 
         NBodySystem bodies(0.01);
-        //std::cout << bodies.energy() << std::endl;
+
+        double energy1 = bodies.energy();
+        if (energy1 > 10) //should always be false
```
Comment (Villiamfj) [inline benchmarks/n-body/cpp/bench.cpp]:
```suggestion
        if (energy1 > 10) //should always be false, but it stops the compiler from removing it
```

File: benchmarks/n-body/cpp/bench.cpp [hunk 5]
```diff
@@ -253,9 +253,21 @@ int main(int argc, char* argv[]) {
...
+        {
+            std::cout << energy1 << std::endl;
+        }
+        
         for (int i = 0; i < n; ++i) bodies.advance();
-        //std::cout << bodies.energy() << std::endl;
+
+        double energy2 = bodies.energy();
+        if (energy2 > 10) //should always be false
```
Comment (Villiamfj) [inline benchmarks/n-body/cpp/bench.cpp]:
```suggestion
        if (energy2 > 10) //should always be false, but it stops the compiler from removing it
```

File: benchmarks/n-body/csharp/Program.cs [hunk 6]
```diff
@@ -218,12 +220,19 @@ public static void RUN(int iterations)
 
 		InitSystem(mem, out V256d* m, out V256d* p, out V256d* v);
 
-		//Console.WriteLine(Energy((double*)mem, p, v).ToString("F9"));
+		double energy1 = Energy((double*)mem, p, v);
+		if (energy1 > 10) //Should always be false
```
Comment (Villiamfj) [inline benchmarks/n-body/csharp/Program.cs]:
```suggestion
		if (energy1 > 10) //Should always be false, but it stops the compiler from removing it
```

File: benchmarks/n-body/csharp/Program.cs [hunk 7]
```diff
@@ -218,12 +220,19 @@ public static void RUN(int iterations)
...
+		{
+			Console.WriteLine(energy1.ToString("F9"));
+		}
 
 		Advance(iterations, 0.01, m, p, v);
 
-		//Console.WriteLine(Energy((double*)mem, p, v).ToString("F9"));
-
+		double energy2 = Energy((double*)mem, p, v);
+		if (energy2 > 10) //Should always be false
```
Comment (Villiamfj) [inline benchmarks/n-body/csharp/Program.cs]:
```suggestion
		if (energy2 > 10) //Should always be false, but it stops the compiler from removing it
```

File: benchmarks/n-body/javascript/bench.js [hunk 8]
```diff
@@ -165,11 +165,20 @@ function N_Body(n) {
 
     offsetMomentum();
 
-    //console.log(energy().toFixed(9));
+
+    let energy1 = energy();
+    if(energy1 > 10){
```
Comment (Villiamfj) [inline benchmarks/n-body/javascript/bench.js]: Comment ligesom forrige

File: benchmarks/n-body/python/bench.py [hunk 9]
```diff
@@ -104,7 +104,9 @@ def report_energy(bodies=SYSTEM, pairs=PAIRS, e=0.0):
         e -= (m1 * m2) / ((dx * dx + dy * dy + dz * dz) ** 0.5)
     for (r, [vx, vy, vz], m) in bodies:
         e += m * (vx * vx + vy * vy + vz * vz) / 2.
-    print("%.9f" % e)
+    
+    if (e > 10):    #Changed
```
Comment (Villiamfj) [inline benchmarks/n-body/python/bench.py]:
```suggestion
    if (e > 10):    #Changed from original. Should always be false, but it stops the compiler from removing it
```

**Review by Villiamfj — CHANGES_REQUESTED** [review]

**Review by Villiamfj — APPROVED** [review]

**Review by Jakobzs — APPROVED** [review]

**Pull request merged**